### ISSUE TITLE
[Squire] Odd annoying behaviour when copy/ pasting anything into an email and then attempting to edit it

### ISSUE THREAD

**Issue by jdaviescoates** [the-djmaze/snappymail#1061]
See https://github.com/the-djmaze/snappymail/issues/1054#issuecomment-1490438032
And https://github.com/the-djmaze/snappymail/issues/1054#issuecomment-1490912208

Creating a new issue as #1054 was prematurely closed and I can't see how to re-open it.

cc'ing @andreasdirscherl

**Comment by the-djmaze** [comment]
Please test with https://github.com/the-djmaze/snappymail/releases/tag/v2.27.3

**Comment by jdaviescoates** [comment]
Thanks, will do as soon as that's available on Nextcloud/ Cloudron!

**Comment by jdaviescoates** [comment]
There are still issues. Pasting that same bit of plain text from a text doc (as shown in the videos linked in the comment linked din the OP) the lines still get muddled up and then deleting the lines between the lines using the Del key still makes other lines move about too. 

In SnappyMail 2.27.3
https://youtu.be/gk6iqBGaMgM

Same thing in Roundcube
https://youtu.be/gU8w7WtaISU

**Comment by jdaviescoates** [comment]
Also, I just copy pasted some text from a PDF and SnappyMail stripped out all the spaces. Pasting the exact same text into Roundcube or a plain text docs and the spaces were all still there

This was the text:

Boombox Music Club with BSharp at 3:45pm – 5.00pm
A creative music club for young musicians age 8-13 to create music together.

**Comment by the-djmaze** [comment]
> copy pasted some text from a PDF and SnappyMail stripped out all the spaces

This depends how the PDF is constructed.
On copy/paste from PDF i even get things like:
```
<span style="left: 637.777px; top: 91.6279px; font-size: 11.0713px; font-family: sans-serif; transform: scaleX(0.997246);" role="presentation" dir="ltr">test here</span>
```

What should we do?
- Remove all HTML and only allow paste as text/plain?
- Remove all CSS?
- Convert weird PDF spaces/dashes/etc. ?

**Comment by the-djmaze** [comment]
Can you test at https://snappymail.eu/demo/

**Comment by the-djmaze** [comment]
Should be resolved with v2.31.0

**Comment by the-djmaze** [comment]
I close this in favor of #1389 where i explain more
